hare/eligibility/oracle_test.go

```diff
@@ -20,8 +20,8 @@ const defSafety = types.LayerID(25)
 const defLayersPerEpoch = 10
 const defNonGenesisLayer = defLayersPerEpoch*2 + 1
 
-var someErr = errors.New("some error")
-var myErr = errors.New("my error")
+var errFoo = errors.New("some error")
+var errMy = errors.New("my error")
```

Comment:
Was easier to read before, IMO. Perhaps `errOne` and `errTwo`?

Reply:
I know, but apparently, GO lint requires that errors will start with err. Anyway, it is just a UT. Nothing critical.